### PR TITLE
Added styles,manifest reference, intent for StartActivity SplashScree…

### PR DESCRIPTION
**Pull request by KonradMiskiewicz**
…n and icons for future DecentWork App

### CODE REVIEW

File: app/src/main/java/com/example/teamyuml/decentworkmobile/SplashScreen.java [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
 
+import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 
 public class SplashScreen extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+
```
Comment (kaniak274) [inline app/src/main/java/com/example/teamyuml/decentworkmobile/SplashScreen.java]: Unneded line.

File: app/src/main/java/com/example/teamyuml/decentworkmobile/SplashScreen.java [hunk 2]
```diff
@@ -1,13 +1,18 @@
 package com.example.teamyuml.decentworkmobile;
 
+import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 
 public class SplashScreen extends AppCompatActivity {
```
Comment (kaniak274) [inline app/src/main/java/com/example/teamyuml/decentworkmobile/SplashScreen.java:12]: Add docstring.

File: app/src/main/res/drawable/splash_background.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@color/colorPrimary" />
+
```
Comment (KamilSupera) [inline app/src/main/res/drawable/splash_background.xml]: Unneded line.

File: app/src/main/res/drawable/splash_background.xml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    <item>
+        <bitmap
+            android:gravity="center"
+        android:src="@mipmap/logo_x1" />
+    </item>
+
```
Comment (KamilSupera) [inline app/src/main/res/drawable/splash_background.xml]: 2xUnneded line.

File: app/src/main/java/com/example/teamyuml/decentworkmobile/SplashScreen.java [hunk 5]
```diff
@@ -1,13 +1,18 @@
...
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+
+        startActivity(new Intent(SplashScreen.this, Login.class));
+        finish();
```
Comment (kaniak274) [inline app/src/main/java/com/example/teamyuml/decentworkmobile/SplashScreen.java]: Should finish(); be in here rather than in the end of method?

**Pull request merged**